src/nav/logs.yml

```diff
@@ -104,3 +79,25 @@ pages:
         path: /docs/logs/logs-context/configure-logs-context-ruby
       - title: APM agent APIs for logs in context
         path: /docs/logs/logs-context/annotate-logs-logs-context-using-apm-agent-apis
+  - title: Log API
+    path: /docs/logs/log-api
+    pages:
+      - title: Introduction to the Log API
+        path: /docs/logs/log-api/introduction-log-api
+      - title: Log event data
+        path: /docs/logs/log-api/log-event-data
+      - title: TCP endpoint
+        path: /docs/logs/log-api/use-tcp-endpoint-forward-logs-new-relic
+  - title: Troubleshooting
+    path: /docs/logs/log-management/troubleshooting
+    pages:
+      - title: No data appears
+        path: /docs/logs/troubleshooting/no-log-data-appears-ui
+      - title: JSON message not parsed
+        path: /docs/logs/troubleshooting/json-message-not-parsed
```

Comment:
@barbnewrelic , this looks correct, but when I clicked on it in my local instance, it generated a 404!! Could you double-check in Gatsby preview?

Reply:
Accidentally left "log management" path. Will delete and doublecheck the taxonomy one last time!